src/ADP.Portal.Api/Services/AdoRestAPIService.cs

```diff
@@ -0,0 +1,61 @@
+﻿using ADP.Portal.Api.Config;
+using ADP.Portal.Api.Models.Ado;
+using ADP.Portal.Core.Ado.Infrastructure;
+using Microsoft.Extensions.Options;
+using System.Net.Http.Headers;
+
+namespace ADP.Portal.Api.Services
```

Comment:
I am not sure why the services are created in the API project ? if it is a business/repostiry layer should move to the core project.  Can we discuss on this?

Reply:
Updated code.